sdk/identity/identity/.gitignore

```diff
@@ -1 +1,2 @@
 src/**/*.js
+!test/assets/cert.pem
```

Comment:
what's in this PEM?

Reply:
It's the literal PEM that I used to record this test. It's no longer associated with my account.

Reply:
Do you know if it gets flagged by credscan?

Reply:
The test recordings should be sanitizing out the client assertion that gets sent which contains the signature. If you do this you can use any arbitrary cert for playback, and not need to use the same cert which was used to record. This will reduce the risk of committing a cert which could still be used to authenticate. We use such "dummy" certs in .NET (both PFX and PEM) and I have not seen them get flagged by credscan.

Reply:
The cert I use is not associated to any account, but it's a valid cert. I could replace it with the same exact cert that .Net uses, if that helps.

Reply:
as long as the cert is not capable of authenticating any account it's fine.